submissions/no pain no gain/no pain no gain.md

```diff
@@ -0,0 +1,78 @@
+
+If you want to read more about removed mechanics there is old fff [Cleanup of mechanics](https://factorio.com/blog/post/fff-266), that will explain in detail decisions behind: pickaxes, furnace heating, burner efficiency and ore hardness.
+
+
+### All your factorio pain in one video...
+You might already saw this video, because it was quite popular on reddit and now is going viral on YouTube, but if not then this video collected (almost) all painful things from factorio like <s>xterminator</s> being killed by a train or looking at your stuck spidertrone, into fast paced compilation.
+
+![](https://www.youtube.com/watch?v=2MnXgqCxMIg)
+
+To make this kind of "cinematic" videos is good to know [factorio console](https://wiki.factorio.com/Console) and [map editor](https://wiki.factorio.com/Map_editor). Because it gives you ability to build big factory in no time, modify researches and for example speed up time. There are many great commands what you can use and whole list is on [factorio wiki](https://wiki.factorio.com/Console). It is also interesting, that Factorio trailers used special prepared [Lua scripts](https://www.lua.org/about.html), like we can read in description: "All the actions, movement and logic in general in the trailer are controlled by the game native modding system (Lua scripting)".
+
+![](https://i.imgur.com/BDoJjko.gif)
+
+Whole necessary [factorio API](https://lua-api.factorio.com/latest/) needed to work with Lua is well described on official website.
+
+![](https://i.imgur.com/q9qlUXa.gif)
+
+With the help of the above-mentioned technics, I've build very interesting machines that are useless in every aspect and their only purpose was to deal maximum amount of pain. There is a small list with explanation.
+I wanted to show cumulative "Train out of fuel" alert and to do this, I simply stopped time in editor and make 10 pointless rails circles. In every circle, I've placed one locomotive and setup schedule for infinite repetitions. As time was stopped, there was no problem to put one more wood to each subsequent locomotive. When everything was ready, I exited editor mode to start time and recorded appearing pop-ups.
+
+![](https://i.imgur.com/i40EZ8C.gif)
+
+Second machine was used to make yt thumbnail and it's just four locomotives connected to one circuit condition and set up to travel straight no matter what. You can also see simple self-timer made out of belts to have time for taking best position.
+
+![](https://i.imgur.com/E2Gatvk.gif)
```

Comment:
Each image will need to be moved to media server by editors and alt-text added for accesibility